src/music21/vfShow.ts

```diff
@@ -621,7 +622,7 @@ export class Renderer {
                         activeBeamGroupNotes.push(n.activeVexflowNote);
                     }
                     if (eighthNoteBeam.type === 'stop') {
-                        const vfBeam = new Vex.Flow.Beam(activeBeamGroupNotes);
+                        const vfBeam = new Vex.Flow.Beam(activeBeamGroupNotes, true);
```

Comment:
Let Vexflow autostem notes, as we do with `Beam.applyAndGetBeams()`. (This should eventually be overridable.)